WooCommerce/Classes/ViewRelated/Products/Edit Product/DefaultProductFormTableViewModel.swift

```diff
@@ -507,5 +527,17 @@ private extension DefaultProductFormTableViewModel {
         static let noPriceWarningTitle =
             NSLocalizedString("Variations without price won’t be shown in your store",
                               comment: "Title of the no price warning row on Product Variation main screen when a variation is enabled without a price")
+
+        // Downloads
```

Comment:
I think it's better understandable
```suggestion
        // Downloadable files
```

Reply:
Got it. Addressed the issue.  💯